### ISSUE TITLE
Fatal Exception

### ISSUE THREAD

**Issue by bowyer-app** [yshrsmz/LicenseAdapter#5]
Hi! @yshrsmz 
Thanks for this beautiful library.

I found this crash under Api  Level 19.
stacktrace paste here.
```
Fatal Exception: android.view.InflateException: Binary XML file line #13: Error inflating class ImageView
       at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:713)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:755)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:492)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:397)
       at net.yslibrary.licenseadapter.LicenseAdapter.onCreateViewHolder(LicenseAdapter.java:51)
       at net.yslibrary.licenseadapter.LicenseAdapter.onCreateViewHolder(LicenseAdapter.java:20)
       at android.support.v7.widget.RecyclerView$Adapter.createViewHolder(RecyclerView.java:5482)
       at android.support.v7.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:4707)
       at android.support.v7.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:4617)
       at android.support.v7.widget.LinearLayoutManager$LayoutState.next(LinearLayoutManager.java:1994)
       at android.support.v7.widget.LinearLayoutManager.layoutChunk(LinearLayoutManager.java:1390)
       at android.support.v7.widget.LinearLayoutManager.fill(LinearLayoutManager.java:1353)
       at android.support.v7.widget.LinearLayoutManager.onLayoutChildren(LinearLayoutManager.java:574)
       at android.support.v7.widget.RecyclerView.dispatchLayoutStep2(RecyclerView.java:3028)
       at android.support.v7.widget.RecyclerView.dispatchLayout(RecyclerView.java:2906)
       at android.support.v7.widget.RecyclerView.onLayout(RecyclerView.java:3283)
       at android.view.View.layout(View.java:14894)
       at android.view.ViewGroup.layout(ViewGroup.java:4644)
       at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1694)
       at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1548)
       at android.widget.LinearLayout.onLayout(LinearLayout.java:1457)
       at android.view.View.layout(View.java:14894)
       at android.view.ViewGroup.layout(ViewGroup.java:4644)
       at android.widget.FrameLayout.layoutChildren(FrameLayout.java:453)
       at android.widget.FrameLayout.onLayout(FrameLayout.java:388)
       at android.view.View.layout(View.java:14894)
       at android.view.ViewGroup.layout(ViewGroup.java:4644)
       at android.widget.FrameLayout.layoutChildren(FrameLayout.java:453)
       at android.widget.FrameLayout.onLayout(FrameLayout.java:388)
       at android.view.View.layout(View.java:14894)
       at android.view.ViewGroup.layout(ViewGroup.java:4644)
       at android.widget.FrameLayout.layoutChildren(FrameLayout.java:453)
       at android.widget.FrameLayout.onLayout(FrameLayout.java:388)
       at android.view.View.layout(View.java:14894)
       at android.view.ViewGroup.layout(ViewGroup.java:4644)
       at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1694)
       at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1548)
       at android.widget.LinearLayout.onLayout(LinearLayout.java:1457)
       at android.view.View.layout(View.java:14894)
       at android.view.ViewGroup.layout(ViewGroup.java:4644)
       at android.widget.FrameLayout.layoutChildren(FrameLayout.java:453)
       at android.widget.FrameLayout.onLayout(FrameLayout.java:388)
       at android.view.View.layout(View.java:14894)
       at android.view.ViewGroup.layout(ViewGroup.java:4644)
       at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:2021)
       at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:1778)
       at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1018)
       at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:5715)
       at android.view.Choreographer$CallbackRecord.run(Choreographer.java:761)
       at android.view.Choreographer.doCallbacks(Choreographer.java:574)
       at android.view.Choreographer.doFrame(Choreographer.java:544)
       at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:747)
       at android.os.Handler.handleCallback(Handler.java:733)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:136)
       at android.app.ActivityThread.main(ActivityThread.java:5274)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:515)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:851)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:667)
       at dalvik.system.NativeStart.main(NativeStart.java)
Caused by java.lang.NumberFormatException: Invalid int: "res/color/abc_secondary_text_material_light.xml"
       at java.lang.Integer.invalidInt(Integer.java:137)
       at java.lang.Integer.parse(Integer.java:374)
       at java.lang.Integer.parseInt(Integer.java:365)
       at com.android.internal.util.XmlUtils.convertValueToInt(XmlUtils.java:122)
       at android.content.res.TypedArray.getInt(TypedArray.java:267)
       at android.widget.ImageView.<init>(ImageView.java:155)
       at android.support.v7.widget.AppCompatImageView.<init>(AppCompatImageView.java:57)
       at android.support.v7.widget.AppCompatImageView.<init>(AppCompatImageView.java:53)
       at android.support.v7.app.AppCompatViewInflater.createView(AppCompatViewInflater.java:102)
       at android.support.v7.app.AppCompatDelegateImplV7.createView(AppCompatDelegateImplV7.java:972)
       at android.support.v7.app.AppCompatDelegateImplV7.onCreateView(AppCompatDelegateImplV7.java:1030)
       at android.support.v4.view.LayoutInflaterCompatHC$FactoryWrapperHC.onCreateView(LayoutInflaterCompatHC.java:44)
       at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:684)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:755)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:492)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:397)
       at net.yslibrary.licenseadapter.LicenseAdapter.onCreateViewHolder(LicenseAdapter.java:51)
       at net.yslibrary.licenseadapter.LicenseAdapter.onCreateViewHolder(LicenseAdapter.java:20)
       at android.support.v7.widget.RecyclerView$Adapter.createViewHolder(RecyclerView.java:5482)
       at android.support.v7.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:4707)
       at android.support.v7.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:4617)
       at android.support.v7.widget.LinearLayoutManager$LayoutState.next(LinearLayoutManager.java:1994)
       at android.support.v7.widget.LinearLayoutManager.layoutChunk(LinearLayoutManager.java:1390)
       at android.support.v7.widget.LinearLayoutManager.fill(LinearLayoutManager.java:1353)
       at android.support.v7.widget.LinearLayoutManager.onLayoutChildren(LinearLayoutManager.java:574)
       at android.support.v7.widget.RecyclerView.dispatchLayoutStep2(RecyclerView.java:3028)
       at android.support.v7.widget.RecyclerView.dispatchLayout(RecyclerView.java:2906)
       at android.support.v7.widget.RecyclerView.onLayout(RecyclerView.java:3283)
       at android.view.View.layout(View.java:14894)
       at android.view.ViewGroup.layout(ViewGroup.java:4644)
       at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1694)
       at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1548)
       at android.widget.LinearLayout.onLayout(LinearLayout.java:1457)
       at android.view.View.layout(View.java:14894)
       at android.view.ViewGroup.layout(ViewGroup.java:4644)
       at android.widget.FrameLayout.layoutChildren(FrameLayout.java:453)
       at android.widget.FrameLayout.onLayout(FrameLayout.java:388)
       at android.view.View.layout(View.java:14894)
       at android.view.ViewGroup.layout(ViewGroup.java:4644)
       at android.widget.FrameLayout.layoutChildren(FrameLayout.java:453)
       at android.widget.FrameLayout.onLayout(FrameLayout.java:388)
       at android.view.View.layout(View.java:14894)
       at android.view.ViewGroup.layout(ViewGroup.java:4644)
       at android.widget.FrameLayout.layoutChildren(FrameLayout.java:453)
       at android.widget.FrameLayout.onLayout(FrameLayout.java:388)
       at android.view.View.layout(View.java:14894)
       at android.view.ViewGroup.layout(ViewGroup.java:4644)
       at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1694)
       at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1548)
       at android.widget.LinearLayout.onLayout(LinearLayout.java:1457)
       at android.view.View.layout(View.java:14894)
       at android.view.ViewGroup.layout(ViewGroup.java:4644)
       at android.widget.FrameLayout.layoutChildren(FrameLayout.java:453)
       at android.widget.FrameLayout.onLayout(FrameLayout.java:388)
       at android.view.View.layout(View.java:14894)
       at android.view.ViewGroup.layout(ViewGroup.java:4644)
       at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:2021)
       at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:1778)
       at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1018)
       at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:5715)
       at android.view.Choreographer$CallbackRecord.run(Choreographer.java:761)
       at android.view.Choreographer.doCallbacks(Choreographer.java:574)
       at android.view.Choreographer.doFrame(Choreographer.java:544)
       at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:747)
       at android.os.Handler.handleCallback(Handler.java:733)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:136)
       at android.app.ActivityThread.main(ActivityThread.java:5274)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:515)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:851)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:667)
       at dalvik.system.NativeStart.main(NativeStart.java)
```

**Comment by bowyer-app** [comment]
You defined ```android:tint="?attr/colorControlNormal" ``` here,
https://github.com/yshrsmz/LicenseAdapter/blob/master/library/src/main/res/layout/license_header.xml#L20
but not specified ```<item name="colorControlNormal">@color/hoge</item>``` at styles.xml

so,we should define ``'<item name="colorControlNormal">@color/color</item>'``
at styles.xml

**Comment by yshrsmz** [comment]
Thanks for reporting!

it turned out that `colorControlNormal` is newly added attribute in Lollipop.
So I changed implementation to use TextView's color to tint arrow icon.

This is fixed, and released as v1.2.1

thanks again!

**Comment by bowyer-app** [comment]
Thank you for your quick fix.
It worked!
